src/config/app-config.ts

```diff
@@ -0,0 +1,9 @@
+import { config } from "dotenv";
+
```

Comment:
follow consistent file naming.
you have used camel case for file naming elsewhere.